src/sql/mysql/create.sh

```diff
@@ -1,8 +1,21 @@
 #!/bin/sh
 
+LOG_TAG="[MYSQL_DB_CREATE]"
+STATUS_SUCCESS="[SUCCESS]"
+STATUS_ERROR="[ERROR]"
+EXIT_ERROR=1
+EXIT_SUCCESS=0
+
+# Check usage
+if [ "$#" -ne 2 ]; then
+  echo "$LOG_TAG $STATUS_ERROR : Illegal number of parameters"
+  echo "Usage: $0 <mysql username> <mysql password>"
+  exit EXIT_ERROR
```

Comment:
Good catch! I forgot to test it apparently! Let me update it. Besides this, any change requests?